src/main/host/syscall/epoll.c

```diff
@@ -105,7 +105,7 @@ SysCallReturn syscallhandler_epoll_ctl(SysCallHandler* sys,
     }
 
     const struct epoll_event* event =
-        memorymanager_getReadablePtr(sys->memoryManager,sys->thread, eventPtr, sizeof(*event));
+        process_getReadablePtr(sys->process,sys->thread, eventPtr, sizeof(*event));
```

Comment:
Looks like you didn't introduce this here, but need a space after `sys->process,`. I'm surprised clang-format didn't catch this.

This same issue occurs many times throughout the diff below. I'm only noting it here, but please fix everywhere.